### DISCUSSION TITLE
QEditor remove link from popup.

### DISCUSSION THREAD

**Discussion by ivankorobchuk123** [quasarframework/quasar#15574] [General - Components / Directives / etc]
How to remove the text in the link when the popup closes?
<img width="781" alt="Screenshot 2023-03-16 at 19 36 17" src="https://user-images.githubusercontent.com/24935631/225734089-2bece5a8-95bc-460f-81af-2789b4f6feb0.png">

**Comment by MilosPaunovic** [comment]
Which text, in the URL field?

**Reply by ivankorobchuk123** [reply]
I don't need to show 'https://' text when I open the popup. I need the input to be empty. How can I do it? Any help please. Thank you
